Chap_API_Security.tex

```diff
@@ -41,7 +41,71 @@ \subsection{\code{PMIx_Get_credential}}
 \begin{codepar}
 pmix_status_t
 PMIx_Get_credential(const pmix_info_t info[], size_t ninfo,
-                    pmix_credential_cbfunc_t cbfunc, void *cbdata)
+                    pmix_byte_object_t *credential)
```

Comment:
Does this call need a set of output `info` and `ninfo` values (`pmix_info_t **results, size_t *nresults`) for the system returned set of values that would have been passed to the callback? From my reading, the `info`/`ninfo` passed into this function are not necessarily the same as those passed to the callback. If they are meant to be the same then it is fine as it is.

Reply:
Per teleconf: The `credential` is the result so the `info` and `ninfo` are not needed.